moped-editor/src/views/projects/projectView/ProjectNameEditable.js

```diff
@@ -63,26 +63,7 @@ const ProjectNameEditable = props => {
     setIsEditing(true);
   }
 
-  const updateProjectNameQuery = gql`
```

Comment:
moved this out of the component to match the separation between queries and components we have elsewhere.